addons/mail/models/mail_link_preview.py

```diff
@@ -38,7 +39,10 @@ def _create_from_message_and_notify(self, message):
         link_previews_by_url = {
             preview.source_url: preview for preview in message.sudo().link_preview_ids
         }
+        ignore_pattern = re.compile(f"{request_url}/(odoo|web)(/|$)") if request_url else None
```

Comment:
So if people with their own prod link odoo, they don't have the right to have previews ^^ ? This should either use some base url, either be done in internal repo...

Reply:
This uses `request.httprequest.url_root` which should be whatever domain requests the link preview, not just odoo.com.

Unless I'm missing something?

Reply:
But why should odoo be ignored for other people ? If they link odoo.com/my-super-job-announce, maybe they want the link preview as it is valid for them.